File: src/Commands/Music/skip.ts
import { Command } from "../../Interfaces";

export const command: Command = {
  name: "skip",
  description: "Skips the current playing music",
  category: "Music",
  aliases: ["s"],
  run: (variables) => {
    variables.client.servers.get(variables.guild_id)?.player.skip(variables);
  }
}

File: src/bot/commands/music/skip.rs
use crate::bot::commands::{Context, Error};

/// Skips the current playing song
#[poise::command(
    slash_command,
    prefix_command,
    aliases("s"),
    category = "Music",
    guild_only
)]
pub async fn skip(ctx: Context<'_>) -> Result<(), Error> {
    let server = get_server!(ctx);

    // TODO: add chack for already stopped bot
    server.player.skip_song().await;

    Ok(())
}
